src/charts.js

```diff
@@ -552,13 +552,13 @@ window.submitForm = function submitForm() {
 
         if (packageNames.length >= 1 && packageNames[0].trim() !== '') {
             formData['package'] = $.map(packageNames, function (packageName) {
-                return packageName.trim();
+                return packageName.trim().toLowerCase();
```

Comment:
It's not fine to use `toLowerCase()` here. While it's true that NPM doesn't allow to register new packages using capital letters, this once was possible, e.g. https://www.npmjs.com/package/Set

Reply:
😱  horrifying